### PR TITLE
Fixing managed disk facts

### PR DESCRIPTION
**Pull request by zikalino**
##### SUMMARY
Fixing missing implementation resource_group parameter handling.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
azure_rm_managed_disk_facts

### CODE REVIEW

**Comment by Fred-sun** [comment]
@yungezz @yuwzho review_need

**Review by yuwzho — APPROVED** [review]

**Comment by yuwzho** [comment]
shipit

File: changelogs/fragments/51781-fixing-managed-disk-facts.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - azure_rm_managed_disk_facts - added missing implementation of listing managed disks by resource group
```
Comment (yuwzho) [inline changelogs/fragments/51781-fixing-managed-disk-facts.yaml:2]: put this into backport rather than devel?

Comment (zikalino) [reply changelogs/fragments/51781-fixing-managed-disk-facts.yaml:2]: @jborean93 mentioned that it's best practice to add changelog fragments with pr in devel already, so I am adding now.

**Review by yungezz — APPROVED** [review]

File: lib/ansible/modules/cloud/azure/azure_rm_managed_disk_facts.py [hunk 2]
```diff
@@ -210,6 +210,19 @@ def list_items(self):
                 results.append(managed_disk_to_dict(item))
         return results
 
+    def list_items_by_resource_group(self):
```
Comment (yungezz) [inline lib/ansible/modules/cloud/azure/azure_rm_managed_disk_facts.py:213]: at least one test?

Comment (zikalino) [reply lib/ansible/modules/cloud/azure/azure_rm_managed_disk_facts.py:213]: @yungezz the test is there. and it just started failing, so this fix is fixing that test

Comment (yungezz) [reply lib/ansible/modules/cloud/azure/azure_rm_managed_disk_facts.py:213]: @zikalino why it didn't failed during time of module merging?

**Pull request merged**